### DISCUSSION TITLE
redmi os

### DISCUSSION THREAD

**Discussion by www1045044461** [xiaojieonly/Ehviewer_CN_SXJ#1491] [Q&A]
从2024.10.25开始，更新1.9.8.9版本之后。
出现能看见本子名称，但是打开之后看不到图片内容的问题。![Screenshot_2024-10-27-04-30-07-400_com.xjs.ehviewer.jpg](https://github.com/user-attachments/assets/ff55bb61-ea60-4934-860f-807b405fd17f)

![Screenshot_2024-10-27-04-30-03-945_com.xjs.ehviewer.jpg](https://github.com/user-attachments/assets/c252e443-c06c-4443-9807-29d1f770a127)

**Comment by BeyondArc** [comment]
> 从2024.10.25开始，更新1.9.8.9版本之后。
> 出现能看见本子名称，但是打开之后看不到图片内容的问题。![Screenshot_2024-10-27-04-30-07-400_com.xjs.ehviewer.jpg](https://github.com/user-attachments/assets/ff55bb61-ea60-4934-860f-807b405fd17f)
> 
> ![Screenshot_2024-10-27-04-30-03-945_com.xjs.ehviewer.jpg](https://github.com/user-attachments/assets/c252e443-c06c-4443-9807-29d1f770a127)
> 
> 

补充：点击阅读的话，会显示获取pToken失败

**Reply by HNPA2018** [reply]
一樣，同問

**Comment by xlj524** [comment]
一样
